gms/api/src/main/idl/com.linkedin.dataplatform.dataPlatforms.restspec.json

```diff
@@ -2,7 +2,7 @@
   "name" : "dataPlatforms",
   "namespace" : "com.linkedin.dataplatform",
   "path" : "/dataPlatforms",
-  "schema" : "com.linkedin.dataplatform.DataPlatformInfo",
+  "schema" : "com.linkedin.dataPlatforms.DataPlatform",
```

Comment:
This isn't backwards compatible, right? Please

1) consider the consequences for users as to how they will be impacted
2) communicate that clearly in the commit description
3) help achieve 2 by following out commit description standards

https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#footer

Reply:
Updated the commit description to call this out.

Reply:
That isn't very descriptive to a lay man. How did it break people specifically? What's the fix?

Reply:
Take 2 : Updated it again. Let me know if it looks good now.